code/lib/core-server/src/withTelemetry.ts

```diff
@@ -83,28 +83,22 @@ export async function sendTelemetryError(
       const error = _error as Error | Record<string, any>;
 
       let storybookErrorProperties = {};
```

Comment:
What's the point of have `storybookErrorProperties` as a let, at all?
Looks like it will always be these 3 keys